### ISSUE TITLE
Questions regarding Team command [1.10]

### ISSUE THREAD

**Issue by RiverC** [LatvianModder/FTBUtilities#236]
Given that I can't find documents specifying how the team commands are supposed to work, I'm submitting this as a question rather than a bug.

One: as a player, when I do /ftb team config, it gives me an unspecified error.

Two: I tried this process for creating a new team and adding another player; this was the result
```
 /ftb team leave
(deletes my team)
/ftb team create new_name
/ftb team invite other_player
Other player sees they have been invited to 'new_name'
Other player types:
/ftb team join new_name
Other player sees
other_player is not a member of this team!
/ftb team join
Missing arguments <ID>

```
I understand on a certain level that to do an action on a team you need to be a member, but it would seem that joining the team should be possible once invited. So it's possible that I've misunderstood the way these commands are supposed to be used. 

Additionally, how am I to use the commands on the server? For example, any permutation of "/ftb team <command>' no matter how well formed always says,  **You must specify which player you wish to perform this action on.** How does one do this? Simply appending the player's name onto the end of what would work in the client still returns that same message.

At the moment I'm just trying to manage making sure we can claim chunks without denying each other access to the chunks we work on together... 

Not sure if this is a bug (or incomplete stuff in 1.10) or I am just not understanding the commands. Thanks!

Version(s):

FTBLib-1.10.2-3.0.0.4
FTBUtilities-1.10.2-3.0.0.3

**Comment by LatvianModder** [comment]
There's a bug where it actually doesn't invite you, that's gonna be fixed soon.
You can add the other person's team as ally for now, and he must add you back (/ftb team add_ally <id>)
/ftb team leave only deletes your team when there's no members left. If you leave a team that has other members then its gonna obviously stay

**Comment by RiverC** [comment]
Thanks. I guessed it was probably the case, given the way invites usually work. What about the commands on the server? Is there a way to run them on a player?

**Comment by LatvianModder** [comment]
You can't because.. server doesnt have a team or anything

**Comment by 0xdade** [comment]
Why is this closed? Was it fixed? It'd be nice to have this feature in place. Thanks :)

**Comment by LatvianModder** [comment]
Again, only because server doesn't 'see' teams. I could add a special command, /... admin_team or smth, that would require you to specify team, but thats not currently planned

**Comment by 0xdade** [comment]
I'm asking if there was a fix so that I could invite players to my team, as opposed to having to add_ally every player's team. I don't need to do any server side commands or anything like that, but /ftb team invite <player> doesn't work, they have no way to accept the invite, so I *have* to do /ftb team add_ally <team_id>. I'm asking if there is a fix for this so that a player can join a team versus having to make their own and add_ally their teammates.

**Comment by LatvianModder** [comment]
Actually that was fixed, in 3.2.x

**Comment by 0xdade** [comment]
Okay thanks! I am running an older version of HermitPack and just wanted to confirm if it was fixed :)
